### PR TITLE
fix 9167

### PR DESCRIPTION
**Pull request by leleOFA**
## Short description

in this task it was insert control of delegation
## List of changes proposed in this pull request

- insert creation delegation
- insert revoke with api

### CODE REVIEW

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaGiuridica/DeleghePGPagoPATest.java [hunk 1]
```diff
@@ -130,6 +130,14 @@ public void nellaSezioneDelegatiDallImpresaSiVisualizzaLaDelegaInStatoDiAttesaDi
         delegatiImpresaSection.controlloEsistenzaDelega(this.datiDelega.get("ragioneSociale").toString());
     }
 
+    @And("Nella sezione Delegati dall impresa si visualizza correttamente una delega in stato di attesa di conferma {string}")
+    public void nellaSezioneDelegatiDallImpresaSiVisualizzaCorrettamenteUnaDelegaInStatoDiAttesaConferma(String ragioneSociale) {
+        logger.info("Si controlla che la delega sia in stato attesa di conferma");
+
+        delegatiImpresaSection.waitLoadDelegatiImpresaPage();
```
Comment (afus0) [inline src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaGiuridica/DeleghePGPagoPATest.java:137]: It doesn't check if the `table` element exist

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaGiuridica/DeleghePGPagoPATest.java [hunk 2]
```diff
@@ -497,4 +505,23 @@ public void siAccettaLaDelegaGruppo(String withGroup) {
...
+        } else {
+            this.logger.error("La delega restituita NON è corretta");
+            Assert.fail("La delega restituita NON è corretta");
+        }
+    }
+
+    @And("Si revoca delega come delegante con api")
+    public void siRevocaDelegaComeDelegantConApi() {
+
+        String tokenExchange = loginPGPagoPaTest.getTokenExchangePGFromFile("delegante");
```
Comment (afus0) [inline src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaGiuridica/DeleghePGPagoPATest.java]: `tokenExchange` is unused

File: src/test/resources/feature/3-destinatario/personaGiuridica/deleghePG/008_115_ricercaDelegheConGruppoPG.feature [hunk 3]
```diff
@@ -9,6 +9,17 @@ Feature:La persona giuridica fa una ricerca per gruppo delle deleghe
...
+    And Si visualizza correttamente la pagina Deleghe sezione Deleghe dell impresa
+    And Creo in background una delega per persona giuridica
+      | accessoCome    | delegante         |
+      | fiscalCode     | LELPTR04A01C352E  |
+      | companyName    | Le Epistolae srl  |
+      | displayName    | Le Epistolae srl  |
+      | person         | false             |
+    And Nella sezione Delegati dall impresa si visualizza correttamente una delega in stato di attesa di conferma "Le Epistolae srl"
+    And Si revoca delega come delegante con api
+    And Nella pagina Deleghe si clicca su Deleghe a carico dell impresa
```
Comment (afus0) [inline src/test/resources/feature/3-destinatario/personaGiuridica/deleghePG/008_115_ricercaDelegheConGruppoPG.feature:23]: This `feature step` too doesn't check if there is the `table` element exist

**Review by afus0 — CHANGES_REQUESTED** [review]

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaGiuridica/DeleghePGPagoPATest.java (hunk 1 above)
Comment (leleOFA) [reply src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaGiuridica/DeleghePGPagoPATest.java:137]: done

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaGiuridica/DeleghePGPagoPATest.java (hunk 2 above)
Comment (leleOFA) [reply src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaGiuridica/DeleghePGPagoPATest.java]: deleted

File: src/test/resources/feature/3-destinatario/personaGiuridica/deleghePG/008_115_ricercaDelegheConGruppoPG.feature (hunk 3 above)
Comment (leleOFA) [reply src/test/resources/feature/3-destinatario/personaGiuridica/deleghePG/008_115_ricercaDelegheConGruppoPG.feature:23]: done

**Review by afus0 — APPROVED** [review]

**Pull request merged**